### PR TITLE
Fix some vulnerabilities and other enhancements

### PR DESCRIPTION
**Pull request by marques-work**
This looks like changes to some random files (and, to be fair, it is). But there's a lot of context in the commit messages.

In summary, this basically does:

1. Upgrade `documentation@14.0.0`
2. Fix a timezone-flaky test
3. Document all necessary browser dependencies in the github actions build; on vanilla ubuntu, the list was insufficient
4. Make linting part of the build and fix lint errors

@NivedhaSenthil could I get a review when you have a moment? I will leave some comments inline to the code for context.

### CODE REVIEW

File: lib/data/devices.js
Comment (marques-work) [inline lib/data/devices.js:526]: iPhone 11 entries were duplicated. This was picked up by `eslint`. Here, I'm deleting the first instances, lexically, so this should be equivalent as JS would have honored the last instance anyway.

File: test/unit-tests/timeField.test.js
Comment (marques-work) [inline test/unit-tests/timeField.test.js:52]: This test fails late night in California. Likely has to do with how `input[type=date[` handles construction from a string vs just passing in an ISO datestring to the Date constructor. I bet the constructor parses the ISO date as midnight UTC.

**Comment by NivedhaSenthil** [comment]
Thanks for taking time to fix this ..I will start reviewing :)

**Comment by zabil** [comment]
@marques-work it is a problem with the node version. I bumped the version to node 18 and got the build to work.

**Comment by marques-work** [comment]
Thanks @zabil!

**Comment by marques-work** [comment]
Whoever sees this next, I think the tests are waiting on approval to run. No rush on it, just don't want it to be missed entirely.

**Review by NivedhaSenthil — APPROVED** [review]

**Comment by NivedhaSenthil** [comment]
LGTM

**Pull request merged**
